include/ddprof_context.hpp

```diff
@@ -43,12 +40,5 @@ struct DDProfContext {
   std::vector<PerfWatcher> watchers;
   ExporterInput exp_input;
   DDProfWorkerContext worker_ctx;
-
-private:
```

Comment:
why remove the deletes ?

Reply:
Several reasons:
- they should not be private, this was the old way of preventing copy
- if there is no user defined destructor, then there is no need for them:
  - either the copy is allowed by members, so why prevent it ?
  - either some member is not copiable, and therefore already makes the class non-copyable